Tests/IPAFlatSteps.fcl

```diff
@@ -1,13 +1,13 @@
 #include "Production/JobConfig/primary/IPAStopsFlat.fcl"
 source.firstRun: 501 #TODO
-physics.filters.IPAStopResampler.fileNames : [
+physics.filters.IPAMuminusStopResampler.fileNames : [
   "/pnfs/mu2e/scratch/datasets/phy-sim/sim/mu2e/IPAStopsCat/MDC2020i/art/6b/10/sim.mu2e.IPAStopsCat.MDC2020i.001202_00000000.art"
   //"root://fndca1.fnal.gov:1094/pnfs/fnal.gov/usr/mu2e/scratch/datasets/phy-sim/sim/mu2e/IPAStopsCat/MDC2020i/art/6b/10/sim.mu2e.IPAStopsCat.MDC2020i.001202_00000000.art"
 ]
 
 physics.producers.generate.startMom : 0
 physics.producers.generate.endMom : 100
-physics.filters.IPAStopResampler.mu2e.MaxEventsToSkip: 4143
+physics.filters.IPAMuminusStopResampler.mu2e.MaxEventsToSkip: 4143
```

Comment:
Please update these collections and the MaxEventsToSkip to correspond to MDC2020r